packages/asc-ui/src/components/Footer/FooterHeading/FooterHeading.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from '@datapunt/asc-core'
+import Heading from '../../Heading'
+import { color, svgFill } from '../../../utils'
+import IconStyle from '../../Icon/IconStyle'
+
+export const FooterHeading = styled(Heading)`
+  margin-top: 0;
+  margin-bottom: 8px;
+  color: ${color('tint', 'level1')};
+  display: flex;
+`
+
+export const ExpandableFooterHeading = styled(FooterHeading)`
```

Comment:
Doesn't this belong to `ToggleFooterHeader`?

Reply:
correct. thanks!